go/autoscaler/autoscaler.go

```diff
@@ -38,9 +38,9 @@ type ClusterResource struct {
 	GPULimit   int
 	GPUTotal   int
 
-	CPURequestKilo float64
-	CPULimitKilo   float64
-	CPUTotalKilo   float64
+	CPURequestMilli int64
```

Comment:
K8s CPU request can be at least `0.1` core which means `100m` (100 milli CPU), change kilo to milli